nx-dev/ui-company/src/lib/co-founders.tsx

```diff
@@ -0,0 +1,93 @@
+import { SectionHeading } from './section-tags';
+import Image from 'next/image';
+import { TwitterIcon } from '@nx/nx-dev/ui-common';
+
+const coFounders = [
+  {
+    name: 'Victor Savkin',
+    title: 'Co-Founder, CTO',
+    image: 'victor-savkin.avif',
+    socials: [
+      {
+        icon: TwitterIcon,
+        url: 'https://x.com/victorsavkin',
+      },
+    ],
+  },
+  {
+    name: 'Jeff Cross',
+    title: 'Co-Founder, CEO',
+    image: 'jeff-cross.avif',
+    socials: [
+      {
+        icon: TwitterIcon,
+        url: 'https://x.com/jeffbcross',
+      },
+    ],
+  },
+];
+export function CoFounders(): JSX.Element {
+  return (
+    <section>
+      <div className="slate-100 border border-y bg-slate-50/40 dark:border-slate-800 dark:bg-slate-800/60">
+        <div className="mx-auto max-w-7xl px-4 py-12 sm:px-6 lg:px-8 lg:py-24">
+          <div className="space-y-12 lg:grid lg:grid-cols-3 lg:gap-8 lg:space-y-0">
+            <div className="space-y-5 sm:space-y-4">
+              <SectionHeading as="h2" variant="title">
+                Meet Our <br /> Co-Founders
+              </SectionHeading>
+            </div>
+            <div className="lg:col-span-2">
+              <ul className="grid grid-cols-2 justify-items-center sm:-mt-8 lg:grid-cols-3 lg:gap-20">
+                {coFounders.map((coFounder, _) => {
+                  return (
+                    <li key={coFounder.name} className="sm:py-8">
+                      <div className="space-y-4">
+                        <div className="aspect-w-3 aspect-h-4">
+                          <Image
+                            width={0}
+                            height={0}
+                            sizes="100vw"
```

Comment:
Note: This `Image` usage also appears to be wrong, [from the Vercel documentation](https://nextjs.org/docs/app/building-your-application/optimizing/images#local-images), not providing `width` and `height` should only set using locally imported images in the `tsx` file which is not the case here.